### ISSUE TITLE
Skip Forward/Backward 20s buttons are missing labels(AV)

### ISSUE THREAD

**Issue by LanieOkorodudu** [UniversalViewer/universalviewer#1735]
**UV version:**
v4.2.1

**I'm submitting a:**

- [x] bug report => please fork one of these [codesandbox examples](https://github.com/UniversalViewer/universalviewer/wiki/UV-Examples) with a repro of your issue and include a link to it below
- [ ] feature request => please use the [user stories repo](https://github.com/UniversalViewer/user-stories)
- [ ] support request => Please do not submit support requests here, use [stackoverflow](http://stackoverflow.com/questions/tagged/iiif)

**Current behavior:**
The video player’s skip forward and skip backward (20 seconds) buttons are currently not labelled.

**Expected behavior:**
The video player’s skip forward and skip backward (20 seconds) controls are currently not labeled.

**Steps to reproduce:**

1. Play this AV manifest: https://iiif.io/api/cookbook/recipe/0064-opera-one-canvas/manifest.json
2. Move your cursor over the player controls.
3. Check the skip forward and skip backward buttons.

**Comment by demiankatz** [comment]
It may be worthwhile to double check whether upgrading MediaElement in #1298 makes a difference to this issue.
